### ISSUE TITLE
Choppy Closing Sequence and Scrolling

### ISSUE THREAD

**Issue by TheLagJag** [DirtDiglett/Pressure2#52]
#### Description of the issue.

When I close the window, it has a choppy fade animation and when i scroll it is a little slow.
#### Does it also occur on Steam without the skin installed?

No.
#### Are you in the Steam BETA? If so, does the issue occur when opted out of the BETA?

No.
#### What Operating System are you using? (Windows 10, Mac OSX 11.1, Linux, Ubuntu etc.)

Windows 10 Home.
#### Are you using the latest stable release or using the latest commit version?

Yes.
#### A screenshot of the issue.

N/A
#### Steps to reproduce the issue.

Scroll and open a window and close it.

Here is a video of scrolling:
[Steam 5_8_2016 4_51_55 PM.zip](https://github.com/DirtDiglett/Pressure2/files/254126/Steam.5_8_2016.4_51_55.PM.zip)

**Comment by SubGum** [comment]
I've been experiencing the same exact issue with all the same conditions.

I've also attached a video of the issue with a side by side of a Chrome window to more easily demonstrate the difference in smoothness:
[Pressure2_Slow_Animations_7-20-16.zip](https://github.com/DirtDiglett/Pressure2/files/374824/Pressure2_Slow_Animations_7-20-16.zip)

I'll add that it's not just the scrolling and closing fade animations. 
Resizing animations of the Steam window are also choppy. 
Basically any animation from the Steam window is choppy while everything else on the PC, or Steam w/ default skin are perfectly smooth.

**Comment by Js41637** [comment]
Honestly I don't know why it does this or what causes it, I don't seem to experience any of these issues. I know the games list can be choppy if you have icons enabled but for me it behaves the same as the default skin.

**Comment by DirtDiglett** [comment]
Yeah, this is definitely a user perception issue. I'm taking a guess in that it's not perceived in the default skin due to the background imagery being fixed on the default skin, whereas it moves with the pressure skin and the bright background makes it much more obvious. The client default skin and pressure side by side scroll exactly the same with the exact same amount of pixels moved. Steam doesn't use 'smooth scrolling' like you'd see in Chrome for example, so when you scroll from A to B, it does a jarring jump, jump, jump, in a set number of pixels/lines each time, which is where this choppiness appears from.
